Fixed: ?? didn't work here as bool won't be seen as null

Quix0r opened the pull request:
`??` didn't catch `false` here, needs to check explicitly because the returned type is always `array`.

PS: I just checked all occurrences. And some needed fixing.

Quix0r committed:
Fixed: ?? didn't work here as bool won't be seen as null

Quix0r committed:
Images::getInfoFromURL[Cached]() will both return empty arrays on error and
that needs to be reflected here, too.

MrPetovan commented on src/Content/Text/BBCode.php:
```diff
@@ -111,7 +111,7 @@ private static function getOldAttachmentData($body)
 					$picturedata = Images::getInfoFromURLCached($matches[1]);
 
-					if ($picturedata) {
+					if (!empty($picturedata)) {
```
Empty array gets casted as false in a condition, what is the purpose of `empty()` here?

Quix0r replied:
I normally don't like this "magic casting" thing and try to make it clean (with no casting). Plus also in other occurrences I found the `empty($array)` calls. Should I then rewrite all to one way?

MrPetovan replied:
It's not wrong, just superfluous regarding the basic capabilities of the language. I'd say leave existing instances alone but don't add any more.

Quix0r replied:
I would like to make each handling of returned variables from `Imagess::getInfoFromURL[Cached]()` the same way, that we know this is everywhere the same and don't have to check each case.

Quix0r replied:
I have now removed `empty()` from each check to make it be the same on all such occurrences.

Quix0r committed:
Converted multiple single-comment (//) to multi-line comment block (/* */)

Quix0r committed:
Changed:
- empty() is maybe superflous here, still I would prefer a code style that is
  written explicitly and not rely on "magic casting"
